### ISSUE TITLE
Cannot seem to remove server edition despite using the QUICK_TPM toggle file 

### ISSUE THREAD

**Issue by iseeghosts** [AveYo/MediaCreationTool.bat#178]
So I used the quick_tpm_toogle.bat file to install tpm patch for an already downloaded iso, but seeing that this would not present me the option to keep my apps, I decided to not go with that option and nw I can't seem to get rid of Windows server edition from downloaded iso even when I rerun the process as specified to remove the TPM patch.

![image](https://user-images.githubusercontent.com/34771316/183918300-e9d2b61c-3a4f-47e1-be89-020c6fdadaf8.png)

![image](https://user-images.githubusercontent.com/34771316/183918465-ef9ea8dd-8d0a-4d9a-88c4-bd49566bc478.png)

![image](https://user-images.githubusercontent.com/34771316/183918605-76011dd5-0437-43b1-a4f9-d538b9a64ab0.png)

![image](https://user-images.githubusercontent.com/34771316/183918628-19bf2c25-ea26-47b1-867a-2c2ff172025e.png)

It seems as if the tpm patch was never removed as the setup still skips through the TPM check even though the cmd windows says otherwise, the outcome is unaffected even if i use the context menu to remove the patch or even RESTART my computer, unmount and remount the ISO file

**Comment by AveYo** [comment]
It obviously can't undo the changes if YOU HAVE THE ISO MOUNTED. Unmount it, make sure it's in a writable location, right-click, Send to - Script. It should toggle between Added and Removed.  
OR
Place the script next to the iso, open a command prompt there (enter `cmd` in explorer address bar) then enter: 
`Quick_11_iso_esd_wim_TPM_toggle.bat Windows11_InsiderPreview_Client_x64_en-us_25179.iso 0`  
_replace name.iso with yours_

**Comment by iseeghosts** [comment]
I've done this, if I don't unmount and try to run the command, it throws error as it should. But the bizarre part even after I deleted the quick toggle file and then downloaded the ISO file from thw microsof's website and then begin, a quick cmd window with the title "get 11 on 'unsupported' PC appears for a microsecond before exiting and then I get Windows server install option. I am out of options to keep my apps and files here now.
![Screenshot (21)](https://user-images.githubusercontent.com/34771316/184365170-aea184fa-8d0c-4a83-969d-a14729aee142.png)
![Screenshot (22)](https://user-images.githubusercontent.com/34771316/184365194-e045e8a5-ddd2-4c4c-8322-78b873d2865e.png)
![Screenshot (23)](https://user-images.githubusercontent.com/34771316/184365211-646a9350-591b-4d9b-a571-77a2e6eb1e35.png)

**Comment by iseeghosts** [comment]
This is a new ISO file but the same version I had downloaded earlier. IS there any way to avoid this?

**Comment by AveYo** [comment]
ffs you have also run the `Skip_TPM_Check_on_Dynamic_Update.cmd`  
run it again and it will stop automatic bypass (for windows update AND setup from iso - see readme)

**Comment by iseeghosts** [comment]
I had to reinstall.
I am thankful that this program exist, but I did that and it didn't work at that time. I re-ran the file (after I unmounted the ISO), the command assured me that the script was disabled. But for some reasons it wasn't, despite the fact that the script was disabled by rerunning the SKIP_TPM_CHECK file.
I am sorry that got you all riled up, but the aforementioned steps didn't help.
I apologized if I wasn't being clear on that
